src/wymeditor/core.js

```diff
@@ -527,6 +584,239 @@ jQuery.extend(WYMeditor, {
     EXTERNAL_MODULES : {}
 });
 
+WYMeditor.DEFAULT_OPTIONS = {
+    html:       "",
+    basePath:   false,
+    wymPath:    false,
+    iframeBasePath: false,
+    jQueryPath: false,
+    skin:       "basic",
+    lang:       "en",
+    direction:  "ltr",
+    customCommands: [],
+
+    // These values will override the default for their respective
+    // `DocumentStructureManager.structureRules`
+    structureRules: {
+        defaultRootContainer:  'p'
+    },
+
+    boxHtml: String() +
+        "<div class='wym_box'>" +
+            "<div class='wym_area_top'>" +
+                WYMeditor.TOOLS +
```

Comment:
Because it includes these variable references, this can't be declared with the rest of the `WYMeditor` attributes.